AlphaWallet/Core/Types/Session.swift

```diff
@@ -41,6 +41,12 @@ class WalletSession {
     }
 }
 
+extension WalletSession: {
+    var capi10Account: CAIP10Account {
```

Comment:
capi10Account -> caip10Account :)

Reply:
> WalletConnectV2Storage 

it is code parts that were in WalletConnectV2Provider, just extracted to separate storage.

Reply:
i guess the same for v1 is needed

Reply:
> And for our own purposes, what do you think of using publishers to replace delegates, like this:

i guess it's great feature, we can move forward to use the same approach somewhere. (but not really sure), all our code base is based on delegate pattern, is it good idea to refactor all of these?

Reply:
Hold on to delegate for a while. I'm a bit worried it's harder to find (it's actually just 1 more place to look for, not so bad). At least it's not necessary yet, right?